No such file or directory: 'node'

当使用密码登陆，在容器运行时报错：

```
[ERROR] 2021 Aug 08 19:47:35 telegram:  [Errno 2] No such file or directory: 'node'
Traceback (most recent call last):
  File "/app/src/tgfrontend/tg.py", line 72, in onFetch
    new = self.feedmgr.fetchNewFeeds(reload)
  File "/app/src/qzonebackend/feed.py", line 121, in fetchNewFeeds
    tmp = self.getFeedsInPage(i + 1, reload)
  File "/app/src/qzonebackend/feed.py", line 87, in getFeedsInPage
    self.qzone.updateStatus(reload)
  File "/app/src/qzonebackend/qzone.py", line 174, in updateStatus
    cookie = self.login()
  File "/app/src/qzonebackend/qzone.py", line 80, in login
    if (r := f()): return r
  File "/app/src/qzonebackend/qzone.py", line 54, in _upLogin
    if r[0] == 0: return t.login(r, all_cookie=True)
  File "/usr/local/lib/python3.9/site-packages/tencentlogin/up/__init__.py", line 123, in login
    'p': self.encodePwd(r),
  File "/usr/local/lib/python3.9/site-packages/tencentlogin/up/__init__.py", line 77, in encodePwd
    return self.getEncryption(self.user.pwd, salt, r[1])
  File "/usr/local/lib/python3.9/site-packages/tencentlogin/up/__init__.py", line 36, in __call__
    p = subprocess.Popen(
  File "/usr/local/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'node'
Stack (most recent call last):
  File "/app/src/__main__.py", line 100, in <module>
    main()
  File "/app/src/__main__.py", line 96, in main
    bot.run()
  File "/app/src/tgfrontend/tg.py", line 140, in run
    self.idle()
  File "/app/src/tgfrontend/tg.py", line 145, in idle
    self.onStart(
  File "/app/src/tgfrontend/tg.py", line 129, in onStart
    self.onRefresh(update, context, reload=self.reload_on_start)
  File "/app/src/tgfrontend/tg.py", line 125, in onRefresh
    self.onFetch(context.bot, reload)
  File "/app/src/tgfrontend/tg.py", line 80, in onFetch
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,5 +14,8 @@ COPY --from=build /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.
 
 WORKDIR /app
 
+RUN mkdir config
+RUN cp misc/example.yaml config/config.yaml
+
 ENTRYPOINT ["python", "src/__main__.py"]
 CMD ["--no-interaction"]

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,8 +4,8 @@ COPY . /app
 
 WORKDIR /app
 
-RUN pip install -e .
-RUN pip install python-telegram-bot[socks]
+RUN pip install -e . && \
+    pip install python-telegram-bot[socks]
 
 FROM python:3.9-alpine
 
@@ -14,9 +14,9 @@ COPY --from=build /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.
 
 WORKDIR /app
 
-RUN mkdir config
-RUN cp misc/example.yaml config/config.yaml
-RUN apk add nodejs
+RUN mkdir config && \
+    cp misc/example.yaml config/config.yaml && \
+    apk add nodejs
 
 ENTRYPOINT ["python", "src/__main__.py"]
 CMD ["--no-interaction"]

--- a/Dockerfile
+++ b/Dockerfile
@@ -16,6 +16,7 @@ WORKDIR /app
 
 RUN mkdir config
 RUN cp misc/example.yaml config/config.yaml
+RUN apk add nodejs
 
 ENTRYPOINT ["python", "src/__main__.py"]
 CMD ["--no-interaction"]

